crates/uv-python/src/installation.rs

```diff
@@ -75,13 +77,10 @@ impl PythonInstallation {
         cache: &Cache,
         preview: Preview,
     ) -> Result<Self, Error> {
-        Ok(find_best_python_installation(
-            request,
-            environments,
-            preference,
-            cache,
-            preview,
-        )??)
+        let installation =
+            find_best_python_installation(request, environments, preference, cache, preview)??;
+        installation.warn_if_outdated_prerelease(request, None);
```

Comment:
Hey @terror and @zanieb, it's problematic to not always pass the `python_downloads_json_url` as it may warn users about versions that are not really available to them, as it does not look at the actual json that they use (if it's configured)

I think it should be passed, or not try to warn at all if `python_downloads_json_url` is configured

Reply:
https://github.com/astral-sh/uv/issues/16711